### PR TITLE
Run ruby 2.5.3 and 2.6.3 on Travis CI

### PR DESCRIPTION
**Pull request by einzige**
Since recent Amazon EB instances are running on 2.6, we need to test the gem against possible failures / deprecations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,7 +157,7 @@ Deprecation means that we discourage the use of a feature, design or practice be
 
 We remove deprecated features after **three months** from the time of announcement.
 
-The security of our customers' assets is of paramount importance to us and sometimes we have to deprecate features because they may pose a security threat or because new, more secure, ways are available. On such occasions we reserve the right to set a different deprecation period which may range from **immediate removal** to the standard **three months**.
```
Comment (einzige) [inline README.md]: trailing space

File: README.md [hunk 2]
```diff
@@ -178,4 +182,4 @@ Copyright (c) 2016-2019 Currencycloud. See [LICENSE][license] for details.
 [semver]:    http://semver.org/
 [license]:   LICENSE.md
 [contr]:     CONTRIBUTING.md
-[hof]:       HALL_OF_FAME.md
\ No newline at end of file
+[hof]:       HALL_OF_FAME.md
```
Comment (einzige) [inline README.md]: no new line at the end of file

File: README.md [hunk 3]
```diff
@@ -178,4 +182,4 @@ Copyright (c) 2016-2019 Currencycloud. See [LICENSE][license] for details.
 [semver]:    http://semver.org/
 [license]:   LICENSE.md
 [contr]:     CONTRIBUTING.md
-[hof]:       HALL_OF_FAME.md
```
Comment (einzige) [inline README.md]: no new line at the end of file

**Comment by EAddario** [comment]
Thank you for contributing @einzige

**Pull request merged**